analytics/src/analytics/metrics/burndown.py

```diff
@@ -105,12 +105,12 @@ def get_stats(self) -> dict[str, Statistic]:
         # get open and closed counts and percentages
         total_opened = int(df["opened"].sum())
         total_closed = int(df["closed"].sum())
-        pct_closed = round(total_closed / total_opened * 100, 2)
+        pct_closed = int(total_closed / total_opened * 100)
```

Comment:
I think we might actually want this: 
```suggestion
        pct_closed = int(round(total_closed / total_opened * 100, 0))
```

`int()` by itself just floors a float and we want to round to the nearest whole number